grafana-plugin/e2e-tests/utils/schedule.ts

```diff
@@ -13,7 +13,7 @@ export const createOnCallSchedule = async (page: Page, scheduleName: string, use
   (await page.waitForSelector('button >> text=Create >> nth=0')).click();
 
   // fill in the name input
-  await fillInInput(page, 'div[class*="ScheduleForm"] input[name="name"]', scheduleName);
```

Comment:
my guess is that the css class names were changed in #3837 (I'm guessing because of the `@emotion/css` dependency [being added](https://github.com/grafana/oncall/pull/3837/files#diff-5c26e441d662e382bbb09a0f0b4e5b18eabf786f1fff0a19500f9c1bffb45699R142)?)

nevertheless, selecting locators based on css class names is fragile and we should prefer `data-testid` whenever possible.

Reply:
Completely agree, selecting by aria roles, text and/or testids is recommended